websockets/websockets.go

```diff
@@ -448,13 +444,15 @@ func (w *webSocket) queueMessage(msg *message) {
 		ev := w.newEvent(events.MESSAGE, msg.t)
 
 		if msg.mtype == websocket.BinaryMessage {
-			if w.binaryType == "" {
-				w.binaryType = arraybufferBinaryType
-				w.vu.State().Logger.Warn(binarytypeWarning)
+			var data any
+			// Use "blob" as default, as per spec:
+			// https://developer.mozilla.org/en-US/docs/Web/API/WebSocket/binaryType
+			if w.binaryType == "" || w.binaryType == blobBinaryType {
```

Comment:
by the specification `binaryType` will be `blob` at the beginning. 

Given that we are breaking the behaviour, I am not certain if we want to have at least one k6 version with it being empty and just throwing an exception instead of warning. 

That was kind of the original plan in my mind to let more people know this will be broken, but to be honest I am not certain it would make much of a difference in people updating their code if they don't update k6 all the time. 

So it might be better to just switch over to `blob` by default either way,

Reply:
Gotcha 👍🏻  I see, so #65 basically